docs/UserGuide.md

```diff
@@ -130,25 +130,36 @@ Edits an existing order in the order list.
 Format: `edito INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/ITEM_NAME] [q/QUANTITY]`
 
 * Edits the order at the specified `INDEX`.
+* This feature is case-insensitive.
 * The index refers to the index number shown in the displayed order list.
 * The index **must be a positive integer** 1, 2, 3, …​
-* Every field is optional, except for `i/ITEM_NAME` and `q/QUANTITY`. Both fields need to be present to update
-  the customer's ordered items.
-* When editing items in the order list, this feature is case-insensitive.
-* You can only edit the order list to consist of items that exists in your inventory. <br> 
+* Every field is optional, but if you were to include `i/ITEM_NAME`, you must also include 
+  `q/QUANTITY`. Both fields need to be present to update the customer's ordered items.
+* You can only edit the customer's ordered items list to consist of items that exists in your inventory. <br> 
   e.g. If your inventory does not have `Apples`, then you cannot edit your customer's order list to have `Apples`.
 * Editing an item that does not exist in your customer's order list, but exists in your inventory 
```

Comment:
Can consider changing `customer's order list` to `a customer's order's list of ordered items` or something similar to disambiguate the meaning here. This can be applied to all the other instances that refer to the ordered items as `order list` or anything along those lines.

Reply:
Right, honestly I also got confused with the wording along the way,  but I will sure to proofread and change these to prevent ambiguity.

Reply:
I think i'll change it to `an order's list of ordered items` instead, as it is less wordy and confusing! thanks for the suggestion!